src/uwtools/drivers/mpas_init.py

```diff
@@ -106,7 +107,7 @@ def namelist_file(self):
         update_values.setdefault("nhyd_model", {}).update(
             {
                 "config_start_time": self._cycle.strftime("%Y-%m-%d_%H:00:00"),
-                "config_run_duration": str_duration,
+                "config_stop_time": stop_time.strftime("%Y-%m-%d_%H:00:00"),
```

Comment:
This definitely looks better - do we need to update the tests to check this or similar though?

Reply:
I could add a test that ensures this is done.